### PR TITLE
Clarify how cron jobs work

### PR DESCRIPTION
**Pull request by gimenete**
- Clarify that the service needs to terminate.
- Clarify the relationship with scheduling libraries.

### CODE REVIEW

File: src/docs/reference/cron-jobs.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 title: Cron Jobs
 ---
 
-Cron Jobs allow you to start a service based on a crontab expression.
+Cron Jobs allow you to start a service based on a crontab expression. This means the service is expected to execute a task, and terminate as soon as that task is finished. Make sure the service doesn't leave any resources open, such as database connections, because otherwise the service won't terminate and Railway wont't start it again until the previous execution has finished.
+
+If you are already using a scheduling library or system in your service such as [node-cron](https://www.npmjs.com/package/node-cron) or [Quartz](http://www.quartz-scheduler.org/), Railway cron jobs are a substitute of them that allows you to safe resources between executions.
```
Comment (pierre-borckmans) [inline src/docs/reference/cron-jobs.md:7]: typo: safe -> save

**Review by pierre-borckmans — APPROVED** [review]
LGTM (just a small typo)

File: src/docs/reference/cron-jobs.md (hunk 1 above)
Comment (gimenete) [inline src/docs/reference/cron-jobs.md:7]:
```suggestion
If you are already using a scheduling library or system in your service such as [node-cron](https://www.npmjs.com/package/node-cron) or [Quartz](http://www.quartz-scheduler.org/), Railway cron jobs are a substitute of them that allows you to save resources between executions.
```

Comment (gimenete) [reply src/docs/reference/cron-jobs.md]: Good catch!

**Pull request merged**